tools/clang/test/CodeGenSPIRV/unary-op.prefix-dec.hlsl

```diff
@@ -0,0 +1,81 @@
+// Run: %dxc -T ps_6_0 -E main
+
+// CHECK: [[v3float_1_1_1:%\d+]] = OpConstantComposite %v3float %float_1 %float_1 %float_1
+
+void main() {
+// CHECK-LABEL: %bb_entry = OpLabel
+    int a, b;
+// CHECK:      [[a0:%\d+]] = OpLoad %int %a
+// CHECK-NEXT: [[a1:%\d+]] = OpISub %int [[a0]] %int_1
+// CHECK-NEXT: OpStore %a [[a1]]
+// CHECK-NEXT: [[a2:%\d+]] = OpLoad %int %a
```

Comment:
Yes. I'm trying to optimize for some cases, but some times it takes too much effort to make other things work with the optimization.